src/js/components/DateTime.js

```diff
@@ -233,7 +233,9 @@ export default class DateTime extends Component {
     if (className) {
       classes.push(className);
     }
-    if (typeof value === 'object') {
+    if (value instanceof Date) {
+      value = moment(value).format(format);
+    } else if ((value !== null) && (typeof value === 'object')) {
```

Comment:
minor comment. should we do something like?

`} else if (value && typeof value === 'object') {`